subworkflows/local/deepvariant_caller.nf

```diff
@@ -5,20 +5,40 @@
 params.deepvariant_options = [:]
 
 include { DEEPVARIANT } from '../../modules/local/deepvariant/main'  addParams( options: params.deepvariant_options )
+include { GLNEXUS } from '../../modules/nf-core/modules/glnexus/main'  addParams( options: params.deepvariant_options )
 
 workflow DEEPVARIANT_CALLER {
     take:
         bam // channel: [ val(meta), path(bam), path(bai) ]
         fasta // path(fasta)
         fai // path(fai)
+        sample // channel: [ sample, sex, phenotype, paternal_id, maternal_id, case_id ]
 
     main:
         DEEPVARIANT ( bam, fasta, fai )
+        file_list = DEEPVARIANT.out.gvcf.collect{it[1]}.toList()
+
+        //retrieve case id for glnexus and store it in a new channel called case_meta
+        sample
+            .first()
+            .map{
+                it ->
+                    new_sample_meta = it.clone()
+                    new_sample_meta.id = new_sample_meta.case_id
+                    [ [ 'id':new_sample_meta.id ] ] }
+            .set {case_meta}
+
+        //Combine case meta with the list of gvcfs
+        case_meta.mix(file_list)
+            .collect()
+            .set { ch_gvcfs }
```

Comment:
```suggestion
        ch_gvcfs = case_meta.combine(file_list)
```

I know [combine](https://www.nextflow.io/docs/latest/operator.html?highlight=mix#combine) returns the cartesian product of two channels with more than 1 element each but I don't think we'll encounter that. Perhaps, the combine operator can be a placeholder to simplify the code a bit until we have smth better?

Reply:
Good suggestion! I tested it and it works!